src/nyc_trees/libs/sql.py

```diff
@@ -25,16 +25,22 @@ def _get_rows(sql, params=[]):
         return cursor.fetchall()
 
 
-def get_user_tree_count(user):
+def get_user_tree_count(user, from_date=None):
```

Comment:
I don't see any calls to this function that pass the `from_date` parameter. Is it necessary?